gnd/src/main/java/com/google/android/gnd/persistence/local/room/relations/ProjectEntityAndRelations.java

```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
-package com.google.android.gnd.persistence.local.room;
+package com.google.android.gnd.persistence.local.room.relations;
```

Comment:
Are relations sufficiently closely related to entities (entity-relational model) that they should be in the entity package?

Reply:
I don't have a good answer. There are only a few relations till now but I expect this package to expand with more feature additions. Please suggest what would be better to do here?

Reply:
Ok with leaving it as-is, though will point out that number of instances should only be a rough proxy or a tie-breaker when scoping; i.e., we should seek to group things conceptually rather than by number.